SpanishWords.Web/Controllers/WordController.cs

```diff
@@ -57,25 +61,36 @@ public IActionResult Add()
                 Text = a.Name,
                 Value = a.Id.ToString()
             });
+
             return View(model);
         }
-
         [HttpPost]
         public IActionResult Add(WordViewModel wordViewModel)
         {
-            
-            if (wordViewModel.Word.Spanish == null || wordViewModel.Word.Spanish.Trim() == "") wordViewModel.Word.Spanish = "<Empty>";
-            if (wordViewModel.Word.English == null || wordViewModel.Word.English.Trim() == "") wordViewModel.Word.English = "<Empty>";
+            if (wordViewModel.Word.Spanish == null || wordViewModel.Word.Spanish.Trim() == "" 
```

Comment:
A może wyciągnąć to do osobnej funkcji i zwracać ostateczny, zbiorowy error message, który potem zwracany jest do loga?

Reply:
Done.